scripts/error-codes/dev-expression-with-codes.js

```diff
@@ -117,16 +117,6 @@ module.exports = function(babel) {
             if (prodErrorId === undefined) {
               // The error cannot be found in the map.
               node[SEEN_SYMBOL] = true;
-              if (process.env.NODE_ENV !== 'test') {
```

Comment:
I'd be happy to. Should we set a NODE_ENV = 'build' when we're building or something?